libs/net/services/Actions/Managers/DataSourceIngestManager`.cs

```diff
@@ -165,11 +165,14 @@ public virtual bool VerifyRunOn(DateTime date, ScheduleModel schedule)
         // No limitation imposed by RunON, so always run.
         if (schedule.RunOn == null) return true;
 
+        // If schedule.RunOn is null this is unreachable, but the compiler requires a default.
+        var scheduled = GetSourceDateTime(schedule.RunOn ?? DateTime.MaxValue);
```

Comment:
`schedule.RunOn` will never be null at this point.

Reply:
That was the reason for the comment. Unless I provide a default I get a compiler error:

/workspaces/tno/libs/net/services/Actions/Managers/DataSourceIngestManager`.cs(169,43): error CS1503: Argument 1: cannot convert from 'System.DateTime?' to 'System.DateTime' [/workspaces/tno/libs/net/services/TNO.Services.csproj]

Reply:
`.Value`

Reply:
That worked. Can you explain why? I thought the only difference between the two is that System.DateTime? is nullable.